### PR TITLE
Add branding footer

### PR DESCRIPTION
**Pull request by redox**

### CODE REVIEW

File: src/lib/DocSearch.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ class DocSearch {
     this.autocomplete = autocomplete(this.input, autocompleteOptions, [{
       source: this.getSource(),
       templates: {
-        suggestion: this.getSuggestionTemplate()
+        suggestion: this.getSuggestionTemplate(),
+        footer: '<div class="ads-footer">Search by <a class="ads-footer--logo" href="https://www.algolia.com/docsearch">Algolia</a></div>'
```
Comment (pixelastic) [inline src/lib/DocSearch.js]: Can't we make the whole footer a link instead? Larger area to click.

Comment (redox) [inline src/lib/DocSearch.js]: Actually that was on purpose, but maybe it could make sense.

Comment (pixelastic) [inline src/lib/DocSearch.js]: So, do you think the little link is enough?

Comment (redox) [inline src/lib/DocSearch.js]: Hmmm, yes?

**Pull request merged**